addons/sale_service/tests/test_project_task_procrument.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp.tests import common
+from datetime import datetime
+
+
+class TestProjectTaskProcrument(common.TransactionCase):
+
+    def setUp(self):
+        super(TestProjectTaskProcrument, self).setUp()
+        self.Product = self.env['product.product']
+        self.SaleOrder = self.env['sale.order']
+        self.SaleOrderLine = self.env['sale.order.line']
+        self.ProcurementOrder = self.env['procurement.order']
+        self.ProjectTask = self.env['project.task']
+        self.uom_id = self.env.ref('product.product_uom_day')
```

Comment:
self.env.ref is return object (not a id). so self.product_uom instead of self.uom_id